hermes-common/build.gradle

```diff
@@ -21,7 +21,8 @@ dependencies {
     compile group: 'org.glassfish.jersey.media', name: 'jersey-media-json-jackson', version: versions.jersey
     compile group: 'org.glassfish.jersey.ext', name: 'jersey-bean-validation', version: versions.jersey
 
-    compile group: 'tech.allegro.schema.json2avro', name: 'converter', version: versions.json2avro
+//    compile group: 'tech.allegro.schema.json2avro', name: 'converter', version: versions.json2avro
+    compile files('./json-avro-converter-0.2.10-SNAPSHOT.jar')
```

Comment:
I'm fine with that change, remember to change this one after new version is released